### PR TITLE
fix: set default preferences to AD and CC

### PR DESCRIPTION
**Pull request by williamrijksen**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -47,7 +47,7 @@
     "prettier": "2.5.1",
     "rollup": "^2.66.1",
     "rollup-plugin-bundle-imports": "^1.5.1",
-    "rollup-plugin-chrome-extension": "^4.0.1-12",
+    "rollup-plugin-chrome-extension": "4.0.1-11",
```
Comment (williamrijksen) [inline package.json:50]: This rollback is necessary because the assets are broken after the introduction of `4.0.1-12`

**Review by nstapelbroek — APPROVED** [review]
Works on my machine 🥳

**Pull request merged**